addons/portal/data/mail_templates.xml

```diff
@@ -6,7 +6,9 @@
             <p>Dear <span t-esc="partner.name"/>,</p>
```

Comment:
nitpick but the `task - [id]` should be `task-[id]` to avoid it being interpreted as a commit hash in some clients.